docs/cloud-native-security/falco-setup.asciidoc

```diff
@@ -0,0 +1,117 @@
+. Click **Detection rules (SIEM)**.
+. Search for a rule called `External Alerts`. Install it if necessary, and enable it.
+
+image::images/falco-external-alerts-rule.gif[A gif showing the steps to enable the External Alerts rule]
+
+[discrete]
+[[ingest-falco-setup-falco]]
+== Configure Falco and Falcosidekick 
+
+You can either:
+
+* <<ingest-falco-setup-falco-vm,Send Falco data to {es} from virtual machines (VMs)>>; or,
+* <<ingest-falco-setup-falco-kubernetes,Send Falco data to {es} from Kubernetes>>.
+
+[discrete]
+[[ingest-falco-setup-falco-vm]]
+=== Configure Falco and Falcosidekick for VMs
+
+Multiple methods for configuring Falco to send data from VMs to {es} are available. This guide uses the https://github.com/falcosecurity/falcosidekick/blob/master/docs/outputs/elasticsearch.md[Falco sidekick on Docker using environment variables] method. 
+
+[discrete]
+=== Configure Falco for VMs:
+
+. Refer to Falco's documentation to https://falco.org/docs/setup/packages/[install Falco on the Linux VMs you wish to monitor]. 
+. Once Falco is installed, update `/etc/falco/falco.yaml` as follows:
+.. Enable JSON output: `json_output: true`
+.. Enable HTTP output, and under `http_output`, for the `url` value, enter the `url:port` where Falcosidekick will listen. For example if Falcosidekick is running on localhost:
++
+```
+http_output:
+  enabled: true
+  url: "http://0.0.0.0:2801/"
+```
+
+[discrete]
+[[falco-config-falco-for-vms]]
+=== Configure Falcosidekick for VMs:
+
+. Refer to Falcosidekick's documentation to https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[install Falcosidekick].
+. Use the https://github.com/falcosecurity/falcosidekick/blob/master/docs/outputs/elasticsearch.md[Falcosidekick on Docker using environment variables] method and set your environment variables as follows:
+.. `ELASTICSEARCH_HOSTPORT`: your {es} endpoint URL, which can be found under **Connection details** on the upper right of the **Integrations** page in {kib}.
+.. `ELASTICSEARCH_INDEX`: the {es} index where you want to store Falco logs. 
++
+IMPORTANT: Your `ELASTICSEARCH_INDEX` value must match `logs-falco.alerts-*`.
++
+.. `ELASTICSEARCH_SUFFIX`: The frequency with which you want the {es} index suffix to change. Either `daily`, `monthly`, `annually`, or `none`. 
+.. `ELASTICSEARCH_APIKEY`: The recommended way to authenticate to {es}, by providing an {kibana-ref}/api-keys.html[API key]. Note that support for this environment variable starts with Falcosidekick version 2.30. You can access the latest version on Falcosidekick's https://hub.docker.com/r/falcosecurity/falcosidekick[Docker Hub].
+.. `ELASTICSEARCH_USERNAME` and `ELASTICSEARCH_PASSWORD`: the username and password for an account on your {es} instance. Authentication using these environment variables is not supported on {ecloud} Serverless.
```

Comment:
Initial capitalization of the descriptions is inconsistent. You could go either way; I prefer sentence case for descriptions like this, so:
```suggestion
.. `ELASTICSEARCH_HOSTPORT`: Your {es} endpoint URL, which can be found under **Connection details** on the upper right of the **Integrations** page in {kib}.
.. `ELASTICSEARCH_INDEX`: The {es} index where you want to store Falco logs. 
+
IMPORTANT: Your `ELASTICSEARCH_INDEX` value must match `logs-falco.alerts-*`.
+
.. `ELASTICSEARCH_SUFFIX`: The frequency with which you want the {es} index suffix to change. Either `daily`, `monthly`, `annually`, or `none`. 
.. `ELASTICSEARCH_APIKEY`: The recommended way to authenticate to {es}, by providing an {kibana-ref}/api-keys.html[API key]. Note that support for this environment variable starts with Falcosidekick version 2.30. You can access the latest version on Falcosidekick's https://hub.docker.com/r/falcosecurity/falcosidekick[Docker Hub].
.. `ELASTICSEARCH_USERNAME` and `ELASTICSEARCH_PASSWORD`: The username and password for an account on your {es} instance. Authentication using these environment variables is not supported on {ecloud} Serverless.
```